### ISSUE TITLE
Use the pyodide webpack plugin

### ISSUE THREAD

**Issue by rth** [grimmer0125/webpack-pyodide-demo#7]
We should start using https://github.com/pyodide/pyodide-webpack-plugin for this example or even maybe have the example in https://github.com/pyodide/pyodide-webpack-plugin and remove this repo altogether.

The only think we need some webpack example  to run in CI.
Any opinion on this @mneil ?

**Comment by ryanking13** [comment]
> even maybe have the example in https://github.com/pyodide/pyodide-webpack-plugin and remove this repo altogether.

I'm +1 for it. Opened [an issue](https://github.com/pyodide/pyodide-webpack-plugin/issues/12) in pyodide-webpack-plugin.

**Comment by mneil** [comment]
Yep, good idea, forgot this repository existed :). I'll follow up on the other plugin issues then come back to this.
